spec/factories/transient_registration.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :transient_registration do
+    trait :has_required_data do
```

Comment:
Might be worth adding a comment saying something like this is a placeholder/keeping the structure in . sync with the registration factory, though at this time the trait is not implemented.